HesoyamHospital/GraphicEditor/Search.xaml.cs

```diff
@@ -24,5 +25,65 @@ private void Search_Button_Click(object sender, RoutedEventArgs e)
             _search.ItemsSource = results;
 
         }
+
+        private void Advanced_Search_Click(object sender, System.Windows.Controls.SelectionChangedEventArgs e)
+        {
+            MapLocation mapLocation = (MapLocation)_search.SelectedItem;
+            if (mapLocation == null) {
+                return;
+            }
+            string name = mapLocation.Name;
+            Global.SearchObjectName = name;
+            string hospital = mapLocation.Hospital;
+            string floor = mapLocation.Floor;
+
+
+            switch (floor)
+            {
+                case "Ground Floor":
```

Comment:
Same thing applies here. It's still hard-coded.
We want a window to appear which will generate it's content by the coordinates we pass it.
We certainly don't want to add additional conditional statements every time a new object(in this case hospital) added.